src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -502,6 +513,7 @@ async def _warn_about_any_files_targets(
     if file_tgts:
         # make it easier for the user to find which targets are problematic by including the alias
         targets = await Get(Targets, Addresses, addresses)
+        # targets = await resolve_targets(**implicitly(addresses))
```

Comment:
Should this have an explanatory comment saying that this is what we want the call to be?

Reply:
This was stale - I was debugging something and forgot to revert. Will take a look and re-run my tests locally to see if this was the issue.